src/main/java/net/mcreator/element/types/Command.java

```diff
@@ -35,9 +35,15 @@
 
 @SuppressWarnings("unused") public class Command extends GeneratableElement {
 
+	public transient final int TYPE_STANDARD;

```

Comment:
The purpose of those is to use them in templates, but right now in templates you check for numerical values. You should either use those in templates, or just do numeric checks and not define those.

See configured_tree.json.ftl for example